tests/memoryview_tests.txt

```diff
@@ -36,6 +36,7 @@ run[.]pyarray
 run[.]sequential_parallel
 run[.]special_methods_T561
 run[.]cythonscope
+run[.]shared_utility
```

Comment:
Can we separate out the memoryview specific tests? It seems an undesired side-effect to exclude the shared module tests just because memoryviews are not available for some reason.